hw3/index.html

```diff
@@ -0,0 +1,193 @@
+				</div>
+			</section>
+			<section class="preview__item">
+				<div class="preview__picture preview__picture-2">
+					<a href="#" class="preview__link">a</a>
+				</div>
+				<div class="preview__title">
+					quia porro venium
+				</div>
+				<div class="preview__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+			</section>
+			<section class="preview__item">
+				<div class="preview__picture preview__picture-3">
+					<a href="#" class="preview__link">a</a>
+				</div>
+				<div class="preview__title">
+					dolorest nuwuam et lobiare
+				</div>
+				<div class="preview__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+			</section>
+		</div>
+	</div>
+	<footer class="main-footer">
+		<div class="main-footer__container">
+			<section class="main-footer__item">
+				<div class="main-footer__title">about us</div>
+				<img src="image/border.png" class="main-footer__border">
+				<div class="main-footer__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+				<div class="main-footer__text">
+					Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et.
+				</div>
+				<button class="main-footer__button main-footer__button_blue">
+					more
+				</button>
+			</section>
+			<section class="main-footer__item">
+				<div class="main-footer__title"> 
+					quick links
+				</div>
+				<img src="image/border.png" class="main-footer__border">
+				<ul class="main-footer__list">
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 1</a>
+					</li>
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 2</a>
+					</li>
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 3</a>
+					</li>
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 4</a>
+					</li>
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 5</a>
+					</li>
+					<li class="main-footer__list-item"> 
+						<a href="#" class="main-footer__list-link">some menu link 6</a>
+					</li>
+				</ul>
+			</section>
+			<section class="main-footer__item">
+				<div class="main-footer__title">contact us</div>
+				<img src="image/border.png" class="main-footer__border">
+				<form class="main-footer__form">
+					<input type="text" class="main-footer__input main-footer__input_placeholder" placeholder="Name" name="user-name">
```

Comment:
main-footer__input main-footer__input_placeholder - you are using all of them everywhere, merge it into one class